web-console/src/views/services-view/services-view.tsx

```diff
@@ -211,6 +224,7 @@ ORDER BY
         curr_size: s.currSize,
         max_size: s.maxSize,
         tls_port: -1,
+        start_time: s.start_time,
```

Comment:
did you add `start_time` to the `/druid/coordinator/v1/servers?simple` response also? If so you should update https://github.com/apache/druid/blob/master/docs/operations/api-reference.md#L496 also, if not then update the line of code above.

Reply:
Good catch, removed it.